modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/data_renderers/ActionsDropdownRenderer.js

```diff
@@ -363,7 +365,13 @@ function ActionsDropdownRenderer({actions, itemData, itemId}) {
 
 	if (loading && !inlineEditingAlwaysOn) {
 		return (
-			<ClayButton disabled displayType="secondary" monospaced small>
+			<ClayButton
+				className="actions-dropdown-renderer-loading-indicator"
+				disabled
+				displayType="secondary"
+				monospaced
+				small
+			>
 				<ClayLoadingIndicator small />
 			</ClayButton>
```

Comment:
We should not place loading indicator in a button. See [in Lexicon](https://liferay.design/lexicon/core-components/loading-indicator/#usage):
> Please don’t use loading indicators inside buttons. If you need a loading indicator for an action triggered by a button, place the loading indicator on one of the sides of the button.

In this case, I think it would be acceptable if we showed loading indicator **instead** of the button. This is basically what the existing code is doing, the button around the indicator is 'just for show'. @drakonux what do you think? What would be the best practice here?

I think this ticket is a good opportunity to make the improvement, and not just patch UI that is not ideal.

Reply:
This makes sense @markocikos, in this case if @drakonux agrees, can I remove all the buttons around the loading indicators? we have three equal cases